vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Space.java

```diff
@@ -53,6 +54,9 @@
     @OneToOne(targetEntity=VSImage.class)
     @NotFound(action=NotFoundAction.IGNORE)
     private IVSImage image;
+    
+    @Transient
+    private Boolean hasIncomingLinks;
```

Comment:
java naming conventions for booleans is that the getter would be "is..." or sometimes "has..." or "can...". This won't be possible with this variable name. In other words, this needs to be renamed along with the getter/setter ;)